apps/SageAccountWeb/src/style/_ProfilePage.scss

```diff
@@ -1,84 +0,0 @@
-#body {
-  .profile-page {
```

Comment:
Remove _ProfilePage.scss, since ProfilePage component was deleted in [this commit](https://github.com/Sage-Bionetworks/synapse-web-monorepo/commit/8577247ce42c1e4d6ce9f6f33ca99dd6e833d8b8)